src/components/forms/SelectOptions/index.js

```diff
@@ -14,15 +15,27 @@ module.exports = React.createClass({
   propTypes: {
     options: React.PropTypes.array.isRequired,
     labelKey: React.PropTypes.string.isRequired,
+    fauxFocus: React.PropTypes.number,
     onSelect: React.PropTypes.func.isRequired
   },
 
+  getDefaultProps: function() {
+    return {
+      fauxFocus: -1
+    };
+  },
+
   getInitialState: function() {
     return {
-      fauxFocus: 0
+      fauxFocus: Math.max(this.props.fauxFocus, 0)
     };
   },
 
+  componentWillReceiveProps: function(nextProps) {
```

Comment:
delete: refactor fauxFocus to parent with callbacks to modify. Child is always told what is being focused.